content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -590,7 +590,7 @@ data and may need to be recreated from scratch.
 
 Workarounds:
 
-* Regularly [back up etcd](https://coreos.com/etcd/docs/latest/admin_guide.html). The
+* Regularly [back up etcd](https://etcd.io/docs/v2.3/admin_guide/). The
```

Comment:
is there a v3 link instead of v2?
k8s currently uses etcd v3.

Reply:
We should not link to the v2.3 etcd docs
https://etcd.io/docs/v3.5/op-guide/recovery/ or https://etcd.io/docs/v3.5/tutorials/how-to-save-database/ should be used